eqcorrscan/tests/lag_calc_test.py

```diff
@@ -170,11 +172,20 @@ def test_lag_calc_api(self):
             detections, self.data, template_names, templates,
             shift_len=0.2, min_cc=0.4, horizontal_chans=['E', 'N', '1', '2'],
             vertical_chans=['Z'], cores=1, interpolate=False,
-            plot=False, npy=False)
+            plot=False, export_cc=False)
         self.assertEqual(len(output_cat), len(detections))
         for event in output_cat:
             self.assertEqual(len(event.picks), len(self.data))
 
+    def test_xcorr_pick_family_export_cc(self):
+        cc_dir = 'cc_exported'
+        xcorr_pick_family(
+            family=self.party[0], stream=self.data, shift_len=0.2, plot=False,
+            interpolate=False, export_cc=True, cc_dir=cc_dir)
+        cc_files = glob.glob(os.path.join(cc_dir, '*.npy'))
```

Comment:
You need to add some kind of assertion to make sure that the files are written. At the moment this doesn't actually check that the files are written. e.g. add a line like:
```python
self.assertEqual(len(cc_files), x)
```
where `x` is the number of files you expect to be written.

Reply:
Hi, Thanks for your comment and guiding me.
I just use parameters of last test.
I don't know how many detections will be write.

Reply:
I added a check.